src/core/cs3iface.py

```diff
@@ -48,6 +48,17 @@ def getuseridfromcreds(token, _wopiuser):
     For the CS3 API case, this is just the token'''
     return token
 
+def getcs3reference(endpoint, filepath):
```

Comment:
A cosmetic change as this is a "private" function, and the argument is a generic reference:
```suggestion
def _getcs3reference(endpoint, fileref):
```